h/services/user.py

```diff
@@ -29,7 +29,13 @@ def __init__(self, default_authority, session):
         self.default_authority = default_authority
         self.session = session
 
-        self._cached_fetch = lru_cache_in_transaction(self.session)(self._fetch)
```

Comment:
We're dropping the LRU cache because that would end up prematurely discarding `User` instances that are being prefetched before rendering annotations?

Are there any situations in our code where we have long-lived transactions and keeping `User` instances around for an arbitrarily long time would be a problem?

Reply:
There's no way to warm up the LRU cache, as it's memoizing method calls by the arguments it is called with. So there's no way of initializing the cache with a bulk query and then read from the cache by only passing in single ids.